utilities/shared_disk.py

```
@@ -253,58 +296,622 @@ def verify_shared_disk_data(
+            backing_uuid = device.backing.uuid
+            break
+    if not backing_uuid:
+        raise ValueError(f"Cannot find backing UUID for shared VMDK '{shared_vmdk_path}' on VM '{owner_name}'")
+
+    serial = backing_uuid.replace("-", "").lower()
+    LOGGER.info(f"Shared disk on '{owner_name}': backing.uuid={backing_uuid}, serial={serial}")
+    return serial
+
+
+def _disable_fast_startup(
+    source_provider: "VMWareProvider",
```

Ruff (0.15.18)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)